compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/declarations/ModuleVarDeclTest.java

```diff
@@ -119,4 +119,12 @@ public void testTopLevelRecoveryWithModuleVarQualifiers() {
     public void testInvalidLocalConstDecl() {
         testFile("module-var-decl/module_var_decl_source_18.bal", "module-var-decl/module_var_decl_assert_18.json");
     }
+
+    @Test
+    public void testSimpleModuleVarReference() {
```

Comment:
We have a separate test class for expression stmts: `ExpressionStatementTest`.
Shall we move these tests to there?

In that class shall we organize the new tests as follows,
1. // Test expression statement
tests with no missing and invalid tokens go here
2. // Test expression statement with recovery
other tests.